### PR TITLE
Fix time series XML again

### PR DESCRIPTION
**Pull request by wschin**
This PR address missed comments in #3546.

### CODE REVIEW

File: docs/samples/Microsoft.ML.Samples/Program.cs [hunk 1]
```diff
@@ -9,19 +10,11 @@ public static class Program
 
         internal static void RunAll()
         {
-            int samples = 0;
```
Comment (wschin) [inline docs/samples/Microsoft.ML.Samples/Program.cs]: Should revert...

File: docs/samples/Microsoft.ML.Samples/Program.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@ internal static void RunAll()
             {
                 var sample = type.GetMethod("Example", BindingFlags.Public | BindingFlags.Static | BindingFlags.FlattenHierarchy);
 
-                if(sample != null)
+                if (sample != null)
```
Comment (wschin) [inline docs/samples/Microsoft.ML.Samples/Program.cs]:
```suggestion
                if(sample != null)
```

File: docs/api-reference/io-time-series-spike.md [hunk 3]
```diff
@@ -1,10 +1,6 @@
...
+There is only one input column.
+The input column must be <xref:System.Single> where a <xref:System.Single> value indicates a value at a timestamp in the time series.
 
-###  Estimator Characteristics 
-|  |  | 
-| -- | -- | 
-| Does this estimator need to look at the data to train its parameters? | No | 
-| Input column data type | Vector of <xref:System.Single> | 
-| Output column data type | 3-element vector of <xref:System.Double> |
+This produces a column typed to a 3-element vector.
```
Comment (natke) [inline docs/api-reference/io-time-series-spike.md]: Same suggestion as above

File: docs/api-reference/io-time-series-change-point.md [hunk 4]
```diff
@@ -1,10 +1,6 @@
...
+There is only one input column.
+The input column must be <xref:System.Single> where a <xref:System.Single> value indicates a value at a timestamp in the time series.
 
-###  Estimator Characteristics 
-|  |  | 
-| -- | -- | 
-| Does this estimator need to look at the data to train its parameters? | No | 
-| Input column data type | Vector of <xref:System.Single> | 
-| Output column data type | 4-element vector of <xref:System.Double> |
+It produces a column typed to a 4-element vector.
```
Comment (natke) [inline docs/api-reference/io-time-series-change-point.md]: Suggest: It produces a column that is a vector with 4 elements.

File: src/Microsoft.ML.TimeSeries/SsaSpikeDetector.cs [hunk 5]
```diff
@@ -189,6 +189,13 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, Dat
     ///
     /// [!include[io](~/../docs/samples/docs/api-reference/io-time-series-spike.md)]
     ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | <xref:System.Single> |
+    /// | Output column data type | 3-element vector of<xref:System.Double> |
```
Comment (shmoradims) [inline src/Microsoft.ML.TimeSeries/SsaSpikeDetector.cs]: add space: of <xref:...

**Pull request merged**